src/routes/+layout.svelte

```diff
@@ -151,7 +151,22 @@
     }
 
     /** vercel effect */
-    async function handle_mouseover() {}
+    let hover_glider_element: HTMLDivElement;
+    let sidebar_buttons: Array<HTMLAnchorElement> = new Array<HTMLAnchorElement>();
+
+    function handle_mouseenter(index: number) {
+        const target = sidebar_buttons[index];
+        
+        if (getComputedStyle(hover_glider_element).getPropertyValue("opacity") === "0") {
+            hover_glider_element.style.transform = `translateY(${target.offsetTop}px)`;
+            setTimeout(() => {
+                hover_glider_element.style.opacity = "100";
+            }, 300);
```

Comment:
This is not the best way to handle this. You should use `transitionend` | `animationend` event for this